src/main/scala/zio/prelude/NonEmptyList.scala

```diff
@@ -309,6 +338,42 @@ sealed trait NonEmptyList[+A] { self =>
   final def tails: NonEmptyList[NonEmptyList[A]] =
     unfold(self)(identity)(_.tailOption)
 
+  @tailrec
+  private def takeHelper[A](n1: Int, nel: NonEmptyList[A], taken: List[A]): List[A] =
```

Comment:
We usually list private methods after all public ones.

Reply:
This method has now been deleted.